{"status": "broken", "name": "Using pattern comprehension in RETURN", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "skip"}, {"name": "feature", "value": "PatternComprehension"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython2"}], "stop": 1578486116149, "historyId": "696ac8f1e2932e0cd29cf77a157d61e2", "start": 1578486116029, "steps": [{"status": "passed", "start": 1578486116029, "stop": 1578486116139, "name": "Given an empty graph"}, {"status": "passed", "start": 1578486116140, "stop": 1578486116143, "name": "And having executed", "attachments": [{"source": "b166d8b9-a405-4ea9-8b17-de3861c73745-attachment.txt", "type": "text/plain", "name": ".text"}]}, {"status": "passed", "start": 1578486116144, "stop": 1578486116146, "name": "When executing query", "attachments": [{"source": "eca7ea55-7045-42e7-a5e4-8ce2d4479c94-attachment.txt", "type": "text/plain", "name": ".text"}]}, {"status": "broken", "attachments": [{"source": "406a601d-6df2-45fa-afc4-6459e28ecce2-attachment.csv", "type": "text/csv", "name": ".table"}], "stop": 1578486116148, "start": 1578486116146, "statusDetails": {"message": "ResponseError: RedisGraph does not currently support pattern comprehension\n", "trace": "  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/Steps.py\", line 84, in step_impl\n    raise exception\n"}, "name": "Then the result should be"}, {"status": "skipped", "start": 1578486116149, "stop": 1578486116149, "name": "And no side effects"}], "statusDetails": {"message": "ResponseError: RedisGraph does not currently support pattern comprehension\n", "trace": "  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/Steps.py\", line 84, in step_impl\n    raise exception\n"}, "uuid": "db5697d0-91cb-45a5-b17b-38b0069caeaa"}